### PR TITLE
shanoir-issue#2081: Tag and Quality tab show message + html fixes

### LINKED ISSUE #2081: Quality tab disapear during study edit
Open a study
Click edit
'Quality' tab disapear

You can also open the quality tab before editing the study:
image

### PR DESCRIPTION
**Pull request by pierrehenri-dauvergne**
Fix #2081 

Go to study
Open **Quality tab**
The message changes if there are quality cards or not
_Before_: can check/uncheck the 'check at import'
_Now_: can't check/uncheck
Click on edit
_Before_: the tab disapeared
_Now_: the tab is displayed, the button 'apply now' is disabled, added an edit button to open the quality card's page

Open **Tags tab**
_Before_: if no tag exist, nothing appears on the page
_Now_: a title and a message saying no tags exist is displayed

### CODE REVIEW

**Comment by jcomedouteau** [comment]
@pierrehenri-dauvergne, the "apply at import" change does not trigger the update button to be enabled.
I know this is a recurring problem on this page, but maybe let's not add one more !

Otherwise, it's cool !

**Comment by pierrehenri-dauvergne** [comment]
@jcomedouteau we talked with Julien and changed a bit the approach. In study details, the edit button is to edit study-related fields (not quality card).
Now, if a user is in quality/tree/bids tab and click 'edit', the user will be redirected toward 'general' tab and quality/tree/bids tab will be greyed out.

File: shanoir-ng-front/src/app/quality-control/quality-control.component.html [hunk 1]
```diff
@@ -12,13 +12,13 @@
 along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
 -->
 
-<span class="subtitle">These are the compatible quality cards that you can apply all over your study.</span>
+<span class="subtitle">{{ message }}</span>
```
Comment (julien-louis) [inline shanoir-ng-front/src/app/quality-control/quality-control.component.html:15]: I think messages should be defined in the template when possible

```<span class="subtitle" *ngIf="qualityCards?.length > 0">...</span>```

File: shanoir-ng-front/src/app/tags/tag.creator.component.html [hunk 2]
```diff
@@ -11,6 +11,7 @@
 You should have received a copy of the GNU General Public License
 along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
 -->
+<span class="subtitle">{{ message }}</span>
```
Comment (julien-louis) [inline shanoir-ng-front/src/app/tags/tag.creator.component.html:14]: same

**Review by julien-louis — CHANGES_REQUESTED** [review]

**Review by julien-louis — APPROVED** [review]

**Pull request merged**